src/Explorer/Panes/UploadItemsPane/index.tsx

```diff
@@ -0,0 +1,220 @@
+import { DetailsList, DetailsListLayoutMode, IColumn, SelectionMode } from "office-ui-fabric-react";
+import React, { ChangeEvent, FunctionComponent, KeyboardEvent, useEffect, useRef, useState } from "react";
+import FolderIcon from "../../../../images/folder_16x16.svg";
+import * as Constants from "../../../Common/Constants";
+import { userContext } from "../../../UserContext";
+import { logConsoleError } from "../../../Utils/NotificationConsoleUtils";
+import { UploadDetails, UploadDetailsRecord } from "../../../workers/upload/definitions";
+import Explorer from "../../Explorer";
+import { getErrorMessage } from "../../Tables/Utilities";
+import { GenericRightPaneComponent, GenericRightPaneProps } from "../GenericRightPaneComponent";
+import { Tooltip } from "./Tooltip";
+
+const UPLOAD_FILE_SIZE_LIMIT = 2097152;
```

Comment:
I know this is copypasta, but it would be nice to suffix with the unit, if you know it (like `UPLOAD_FILE_SIZE_LIMIT_KB`).

Reply:
done